src/index.html

```diff
@@ -1,13 +1,427 @@
+                      class="art__painting"
+                    >
+                  </div>
+                </div>
+
+              </div>
+            </div>
+            <div class="description">
+              <p class="art__name-art">
+                Bodhisattvas of Wisdom, Compassion, and Power
+              </p>
+              <p class="art__description">
+                March 27, 2021 – October 30, 2022
+              </p>
+            </div>
+          </div>
+
+
+          <div class="gallery__art art">
+            <div class="art__old-masters">
+              <div class="grid">
+                <div class="grid__item grid__item-pad--1-2 grid__item-desktop--1-4">
+                  <div class="art__painting-container">
+                    <img 
+                        src="../src/images/photos/3.1.png" 
+                        alt="art 1" 
+                        class="art__painting art__painting-3-1--position"
+                      >
+                  </div>
+                </div>
+                <div class="grid__item grid__item-pad--3-6 grid__item-desktop--5-12">
+                  <div class="art__painting-container">
+                    <img
+                      src="../src/images/photos/3.2.png"
+                      alt="art 1"
+                      class="art__painting art__painting-3-2--position"
+                    >
+                  </div>
+                </div>
+              </div>
+            </div>
+            <div class="description">
+              <p class="art__name-art">
+                Inspiring Walt Disney: The Animation of French Decorative Arts
+              </p>
+              <p class="art__description">
+                December 10, 2021 – March 6, 2022
+              </p>
+            </div>
+          </div>
+
+          <div class="grid grid--desktop grid--pad">
+            <div class="grid__item grid__item-desktop--5-8 grid__item-pad--2-5">
+              <a href="#" class="button">View all exhibitions</a>
+            </div>
+          </div>
+          
+        </div>
+      </section>
+
+      <section class="online-museum page__section" id="testimonials">
+        <div class="container">
+          <h2 class="page__section-title page__section-title--margin">More to Explore<br>Online</h2>
+
+          <div class="grid grid--desktop grid--pad grid--row-online">
+            
+            <div class="online-card grid__item grid__item-row-1 grid__item-desktop--1-4 grid__item-pad--2-5">
+              <div class="online-card__photo-container">
+                <img 
+                src="./images/photos/4.1.png" 
+                alt="More to Explore"
+                class="online-card__photo"
+              >
+              </div>
+              
+              <h3 class="online-card__name">Collection Areas</h3>
+              <p class="online-card__description">These collection areas study, exhibit, and care for the objects in The Met collection.</p>
+            </div>
```

Comment:
Text here should be centered
![image](https://user-images.githubusercontent.com/37161315/170651218-4e976d00-2fed-4354-877c-e4af00d49754.png)

Reply:
Also, checklist 8th item